interactive/templates/interactive/analysis_request_output.html

```diff
@@ -7,57 +7,108 @@
+            {% else %}
+              Analysis Output
+            {% endif %}
           </p>
         </div>
-        <p class="max-w-3xl mx-auto mt-5 text-xl leading-normal text-slate-700">
+        <p class="max-w-3xl mx-auto mt-5 text-lg leading-normal text-slate-700">
           The aim of this OpenSAFELY Interactive report is to describe trends and variation in the recording of clinical codes in health records to inform further analyses.
           It is not intended to be a tool for "performance monitoring".
         </p>
-        <div class="my-4">
-          {% if user.is_staff %}
-            {% url 'request_analysis_email' analysis.id as email_url %}
-            {% if analysis.complete_email_sent_at %}
-              <p class="text-slate-500">"Analysis request is ready" email was sent {{ analysis.complete_email_sent_at | naturaltime }}</p>
-              {% include "components/button.html" with el="link" size="md" color="oxford" text="Resend email to requester" href=email_url %}
-            {% else %}
-              {% include "components/button.html" with el="link" size="md" color="oxford" text="Email to requester" href=email_url %}
-            {% endif %}
-          {% endif %}
-        </div>
       </div>
-      <div class="max-w-3xl mx-auto rounded-md bg-blue-50 border border-blue-300 p-4 pt-6 pb-7 mt-6">
+      <div class="max-w-3xl mx-auto rounded-md bg-slate-50 border border-slate-300 p-4 pt-6 pb-7 mt-6">
         <div class="flex">
           <div class="flex-shrink-0">
-            {% include "icons/outline-information-circle.svg" with html_class="h-6 w-6 text-blue-800" %}
+            {% include "icons/outline-information-circle.svg" with html_class="h-6 w-6 text-slate-800" %}
           </div>
-          <div class="ml-3 font-semibold text-blue-800">
+          <div class="ml-3 font-semibold text-slate-800">
             <p>We would like to hear your feedback about this report. Please take a moment to complete our {% include "components/link.html" with href="https://docs.google.com/forms/d/e/1FAIpQLScWikDx0UlqbEcnbzZhn5FiBTBHc2LtrfhqmmKwgOuKt4oFTQ/viewform" content="feedback form" %}.</p>
           </div>
         </div>
       </div>
-      <div class="max-w-3xl mx-auto rounded-md bg-yellow-50 border border-yellow-300 p-4 pt-6 pb-7 mt-6">
-        <div class="flex">
-          <div class="flex-shrink-0">
-            {% include "icons/solid-exclamation.svg" with html_class="h-6 w-6 text-yellow-500" %}
-          </div>
-          <div class="ml-3 font-semibold text-yellow-800">
-            <p class="mb-3">Sharing outside of NHS England is not permitted in the current pilot phase without further approvals.</p>
-            <p>Please email team@opensafely.org if you would like to seek approval for wider sharing.</p>
-          </div>
-        </div>
-      </div>
     </div>
+
+    {% endif %}
   </header>
 
   <div class="container pt-8">
+    {% if not user.is_staff %}
```

Comment:
Would it be easier to show this to all users? No harm in reminding admins of this too and it reduces the number of conditionals in this template.

Reply:
👍 will change that